### DISCUSSION TITLE
Model default integer value

### DISCUSSION THREAD

**Discussion by joelpanor** [strapi/strapi#6457] [Legacy Topics]
Hi,

I try to set 0 default value in model attribute definition but it doesn't work

I use Mysql Database.

example : `"nbre_point": {
      "type": "integer",
      "default": 0
    },`

After schema's update, the default value is set to NULL.

Can you help me please?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
